babel/messages/checkers.py

```diff
@@ -65,9 +65,6 @@ def _validate_format(format: str, alternative: str) -> None:
     arguments are not interchangeable as `alternative` may contain less
     placeholders if `format` uses named placeholders.
 
-    The behavior of this function is undefined if the string does not use
```

Comment:
I think this is wrong - `python_format` happily calls this method when `format` does not contain any format placeholders (for example when `msgid` contains them but `msgid_plural` does not, `python_format` will still call `_validate_format(msgid, msgstr)`).